snapcraft/tests/test_pluginhandler.py

```diff
@@ -1345,7 +1345,7 @@ def test_clean_stage(self):
             with self.subTest(key=key):
                 self.clear_common_directories()
 
-                schema = {'snap': {'type': 'array'}}
+                schema = {'stage': {'type': 'array'}}
```

Comment:
A copy-paste error in a test that was only discovered as a result of the new `additionalProperties: false`.